### PR TITLE
docs: README for RadioButtonField component

### PR DESCRIPTION
**Pull request by burakukula**
Hey F36!

Just some docs around RadioButtonField component. 🙂

## PR Checklist

- [ ] I have read the relevant `readme.md` file(s)
- [ ] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Storybook stories are added/updated/not required
- [ ] Usage notes are added/updated/not required
- [ ] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [ ] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/forma-36-react-components/src/components/RadioButtonField/README.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+Use RadioButtonField to give users the possibility to select only one option from a number of choices. Radio is generally displayed in a radio group. It contains a radio input with its label. If needed the component can display an error message and hint message. For a multiple-choice, please consider using [ChecboxField component](https://f36.contentful.com/components/checkbox-field/).
```
Comment (denkristoffer) [inline packages/forma-36-react-components/src/components/RadioButtonField/README.mdx:1]: > Radio is generally displayed in a radio group

Is there a word missing here?

File: packages/forma-36-react-components/src/components/RadioButtonField/README.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+## Table of contents
+
+- [How to use RadioButton](#how-to-use-radioButton)
```
Comment (denkristoffer) [inline packages/forma-36-react-components/src/components/RadioButtonField/README.mdx:5]: RadioButtonField

**Review by denkristoffer — CHANGES_REQUESTED** [review]

File: packages/forma-36-react-components/src/components/RadioButtonField/README.mdx (hunk 2 above)
Comment (burakukula) [reply packages/forma-36-react-components/src/components/RadioButtonField/README.mdx]: oh! good catch! thx! Updated.

File: packages/forma-36-react-components/src/components/RadioButtonField/README.mdx (hunk 1 above)
Comment (burakukula) [reply packages/forma-36-react-components/src/components/RadioButtonField/README.mdx]: I actually got to the conclusion, that this sentence is not really needed.  It is explained in "How to use" section.

File: packages/forma-36-react-components/src/components/RadioButtonField/README.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
+Use RadioButtonField to give users the possibility to select only one option from a number of choices. It contains a radio input with its label. If needed the component can display an error message and hint message. For a multiple-choice, please consider using [ChecboxField component](https://f36.contentful.com/components/checkbox-field/).
+
+## Table of contents
+
+- [How to use RadioButtonField](#how-to-use-radioButtonField)
+- [Code examples](#component-variations)
+- [Content recommendations](#content-recommendations)
+- [Accessibility](#accessibility)
+
+## How to use RadioButton
```
Comment (denkristoffer) [inline packages/forma-36-react-components/src/components/RadioButtonField/README.mdx:10]: This too I think, RadioButtonField?

File: packages/forma-36-website/src/pages/components/radio-button-field/index.mdx [hunk 4]
```diff
@@ -5,18 +5,3 @@ status: 'stable'
 github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/RadioButton'
 storybook: 'https://f36-storybook.contentful.com/?path=/story/components-radiobutton--default'
```
Comment (denkristoffer) [inline packages/forma-36-website/src/pages/components/radio-button-field/index.mdx:6]: Looks like these links are to the RadioButton component.

**Review by denkristoffer — CHANGES_REQUESTED** [review]

File: packages/forma-36-react-components/src/components/RadioButtonField/README.mdx (hunk 3 above)
Comment (burakukula) [reply packages/forma-36-react-components/src/components/RadioButtonField/README.mdx]: oh yeah! sorryyyy. updated

File: packages/forma-36-website/src/pages/components/radio-button-field/index.mdx (hunk 4 above)
Comment (burakukula) [reply packages/forma-36-website/src/pages/components/radio-button-field/index.mdx]: yup, thx! updated

**Review by denkristoffer — APPROVED** [review]

**Review by gui-santos — APPROVED** [review]

**Pull request merged**